frontend/src/components/site/Footer.tsx

```diff
@@ -11,10 +11,11 @@ interface FooterProps {
   heading: string;
   items: FooterItem[];
   alignment: "left" | "center" | "right";
```

Comment:
I added `center` to this section. (https://github.com/openmultiplayer/web/actions/runs/8544090420/job/23409356584)
Is it gonna work?

Reply:
i guess, you should try